internal/generator/vector/output/loki/loki_conf_test.go

```diff
@@ -659,105 +859,107 @@ var _ = Describe("Generate vector config for in cluster loki", func() {
 	var f = func(clspec logging.CollectionSpec, secrets map[string]*corev1.Secret, clfspec logging.ClusterLogForwarderSpec, op generator.Options) []generator.Element {
 		return Conf(clfspec.Outputs[0], inputPipeline, secrets[constants.LogCollectorToken], generator.NoOptions)
 	}
-	DescribeTable("for Loki output", helpers.TestGenerateConfWith(f)) //		Entry("with default logcollector bearer token", helpers.ConfGenerateTest{
-	//			CLFSpec: logging.ClusterLogForwarderSpec{
```

Comment:
Remove me if not required

Reply:
this code was commented by mistake in previous commit